content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -9,36 +9,45 @@ This page describes authentication for Kubeflow Pipelines to GCP.
 ## Before you begin
 [Installation Options for Kubeflow Pipelines](/docs/pipelines/installation/overview/) introduces options to install Pipelines. Be aware that authentication support and cluster setup instructions will vary depending on the option you installed Kubeflow Pipelines with.
 
+## Compute Engine default service account
+
+This is good for trying out Kubeflow Pipelines, because it is easy to set up, but not secure.
+
+By default, your GKE nodes use [Compute Engine default service account](https://cloud.google.com/compute/docs/access/service-accounts#default_service_account). If you allowed all scopes when creating the cluster,
+Kubeflow Pipelines can authenticate to GCP with editor permission in your project without further setup.
+
+### Authoring pipelines to use default service account
+
+Pipelines don't need any specific changes to authenticate to GCP, it will use the node's GSA transparently.
```

Comment:
Do you mind elaborating what GSA means here? Does it refer to the default compute engine service account?

Reply:
The node's GSA defaults to compute engine service account, but it could also be different depending on user setup.

I don't want to include too much information here because of complexity in node service account setups.

Do you have any suggestions on the wording change that can reduce some confusion?

Reply:
I've remoed node service account wordings to be consistent with the section's goal of introducing default service account.

Reply:
Looks good. Thanks